Makefile

```diff
@@ -129,9 +130,9 @@ clean:
 
 .PHONY: format
 format:
-	# pip install isort==5.7.0
+	echo "isort, version `isort --version-number`"
 	isort -m VERTICAL_HANGING_INDENT --skip-gitignore --ac --tc --profile black ./baselines/marl_benchmark/ ./cli ./envision ./examples/ ./utils/ ./scenarios/ ./smarts ./ultra ./zoo
-	# pip install black==20.8b1
+	black --version
```

Comment:
If we are not depended on a particular version of black, then we should remove the version dependency from dev field in `setup.py` .

Reply:
We are currently dependent on a specific version of `black` but the intention here is to log that rather than comment it in. The version should only be indicated in the `setup.py` and `requirements.txt` if at all.